setup.py

```diff
@@ -1641,7 +1636,8 @@ def detect_modules(self):
         if TEST_EXTENSIONS:
             self.detect_test_extensions()
         self.detect_readline_curses()
-        self.detect_crypt()
+        if not VXWORKS:
```

Comment:
Nitpick: I would prefer to move this condition *inside* test_crypt(). Moreover, I would prefer to have a short comment explaining why it's not build. For example:

```
def detect_crypt():
    if VXWORKS:
        # bpo-31904: crypt() function is not provided by VxWorks
        return

    ...
```